clang/test/dpct/asm_lop3.cu

```diff
@@ -30,7 +30,8 @@ static __device__ __forceinline__ uint32_t LOP3LUT_ANDOR(uint32_t a, uint32_t b)
 #define B 3
 __device__  int hard(int a) {
   int d4;
-  // CHECK: d4 = (~((a + B) * (a + B)) & B & ~(3)) | (~((a + B) * (a + B)) & B & (3)) | (((a + B) * (a + B)) & ~B & ~(3));
-  asm("lop3.b32 %0, %1 * %1, %2, 3, 0x1C;" : "=r"(d4) : "r"(a + B), "r"(B));
+  // CHECK: d4 = (~(a + B) & B & ~3) | (~(a + B) & B & 3) | ((a + B) & ~B & ~3);
```

Comment:
why update this test case.

Reply:
This case is ill-formed, and rejected by nvcc, as far as I know, the asm syntax does not allow addition of numeric constants and register variables.